services/azure-devops/azure-devops-build.service.js

```diff
@@ -58,22 +65,60 @@ module.exports = class AzureDevOpsBuild extends BaseSvgScrapingService {
         pattern: ':organization/:projectId/:definitionId/:branch',
         namedParams: {
           organization: 'totodem',
-          projectId: '8cf3ec0e-d0c2-4fcd-8206-ad204f254a96',
+          projectId: 'Shields.io',
           definitionId: '2',
           branch: 'master',
         },
         staticPreview: renderBuildStatusBadge({ status: 'succeeded' }),
         keywords,
         documentation,
       },
+      {
+        title: 'Azure DevOps builds (stage)',
+        pattern:
+          ':organization/:projectId/:definitionId/:branch*\\?stage=:stage',
+        namedParams: {
+          organization: 'totodem',
+          projectId: 'Shields.io',
+          definitionId: '5',
+          branch: 'master',
```

Comment:
Maybe it's worth dropping the `branch` for these two new examples?

Reply:
I hesitated to keep branch or not... But if you too think that it would be better, let's drop it indeed.